crates/rolldown/src/utils/renamer.rs

```diff
@@ -144,11 +144,12 @@ impl<'name> Renamer<'name> {
       stack: &mut Vec<Cow<FxHashMap<Rstr, u32>>>,
       canonical_names: &mut FxHashMap<SymbolRef, Rstr>,
     ) {
-      let bindings = module.scope.get_bindings(scope_id);
+      let mut bindings = module.scope.get_bindings(scope_id).iter().collect::<Vec<_>>();
+      bindings.sort_unstable_by_key(|(_, symbol_id)| *symbol_id);
```

Comment:
`get_bindings` no longer returns an indexmap (order keys by insertion order), so we need to sort here.